### PR TITLE
Documents Update & Agent v6.5.1 Release Notes

### PR DESCRIPTION
**Pull request by ywang-nr**
* What problems does this PR solve? 
    - [Mobile-7449] A new doc to clarify a known issue
    - [Mobile-7472] Update an existing doc
    - [Mobile-7473] Agent v6.5.1 release notes

### CODE REVIEW

**Comment by ywang-nr** [comment]
@cthomas-newrelic for review

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/firebase-incompatibility-android.mdx
Comment (cthomas-newrelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/firebase-incompatibility-android.mdx:15]: Typo

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/firebase-incompatibility-android.mdx
Comment (cthomas-newrelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/troubleshoot/firebase-incompatibility-android.mdx:19]: This is not completely true: for the most part Firebase and our agent co-exist just fine. Keep in mind that FB is made up of many packages and I've found it is on the the Performance SDK we have trouble with, and with that only certain features that involve class rewriting. The goal of the story was to isolate and call-out what we can't work with. However, it's probably better at the moment to simply say, "_use one or the other, not both_".

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

**Comment by ywang-nr** [comment]
@dmurray for support team to review

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-651.mdx
Comment (cthomas-newrelic) [inline src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-651.mdx:10]: Please add a reference to where that leak occurred, and what was leaked

**Review by cthomas-newrelic — CHANGES_REQUESTED** [review]

File: src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-651.mdx
Comment (ywang-nr) [reply src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-651.mdx]: Provided more details

**Comment by ywang-nr** [comment]
This PR is ready for review, please let me know before your merge. Thank you.

**Comment by ywang-nr** [comment]
** DO NOT MERGE ** Thank you

**Pull request closed without merging**
